README.md

```diff
@@ -190,13 +177,42 @@ Then you just have to invoke `swiftgen config run`, or even just `swiftgen` for
 
 To learn more about the configuration file — its more detailed syntax and possibilities, how to pass custom parameters, using `swiftgen config lint` to validate it, how to use alternate config files, and other tips — [see the dedicated documentation](Documentation/ConfigFile.md).
 
+There are also more subcommands not related to generating code but more oriented for help and other tasks, namely:
+
+* The `swiftgen templates` subcommands helps you print, duplicate, find and manage templates (both bundled and custom)
+* You can use `--help` on `swiftgen` or one of its subcommand to see the detailed usage.
+
+
+### Directly invoking a subcommand
```

Comment:
Maybe we can hide that in a `<details>…</details>` collapsible section? (Yeah, like we're kinda ashamed of this now that we have the config file option 😄 )